src/NuGet.Indexing/NuGetAcronymExpansionProvider.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace NuGet.Indexing
+{
+    public class NuGetAcronymExpansionProvider : IAcronymExpansionProvider
```

Comment:
create a private constructor so this class cannot be initialized in any other way